http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java

```diff
@@ -1,11 +1,14 @@
 package io.quarkus.ts.http.advanced.reactive;
 
+import org.junit.jupiter.api.Tag;
+
 import io.quarkus.test.bootstrap.KeycloakService;
 import io.quarkus.test.bootstrap.RestService;
 import io.quarkus.test.scenarios.QuarkusScenario;
 import io.quarkus.test.services.KeycloakContainer;
 import io.quarkus.test.services.QuarkusApplication;
 
+@Tag("fips-incompatible") // native-mode
```

Comment:
This fails in JVM CI checks. I didn't follow today's discussions closely, but could that have something to do with Keycloak 18.0.0 vs. 18.0.2?

Also, this shlould make sure the tests are skipped on FIPS machine, so do we still need the `-Dcom.redhat.fips=false` Java option (see a few lines bellow; this also applies to `HttpAdvancedIT`)?